cw_bitcoin/lib/electrum_wallet.dart

```diff
@@ -3193,19 +3195,12 @@ Future<void> _handleScanSilentPayments(ScanData scanData) async {
               // So, if blockDate exists, reuse
               if (isDateNow) {
                 try {
+                  final rootURL = "https://cake.mempool.space";
                   final tweakBlockHash = await ProxyWrapper()
-                      .get(
-                        clearnetUri: Uri.parse(
-                          "https://mempool.cakewallet.com/api/v1/block-height/$tweakHeight",
-                        ),
-                      )
+                      .get(clearnetUri: Uri.parse("$rootURL/api/block-height/$tweakHeight"))
```

Comment:
why `/v1` is removed? is the api working fine?

Reply:
Yes the API is working fine without v1